vitest.config.ts

```diff
@@ -20,6 +21,8 @@ const includeWithStyles = [
 const isReact17 = version.startsWith("17");
 
 export default defineConfig({
+  // TODO: instead of this, we should just make it compatible with config
+  plugins: process.env.ARIAKIT_TEST_LOADER === "solid" ? [solid()] : [],
```

Comment:
Could you clarify this comment? What does "make it compatible with config" mean exactly?

Reply:
As in, actually only compile Solid files without breaking React ones, so that we can have this plugin always enabled instead of this conditional.

Reply:
Is this even possible?

Reply:
It should be, but I think it's broken in the vite plugin. I might actually contribute a few fixes myself, and help Ryan and the Solid core team a bit in that area.

Currently, the community is using hacky workarounds to make those kinds of setups work.